src/application/redux/containers/address-amount.container.ts

```diff
@@ -1,18 +1,19 @@
 import { connect } from 'react-redux';
-import { MainAccountID } from '../../../domain/account';
 import { assetGetterFromIAssets } from '../../../domain/assets';
-import { RootReducerState } from '../../../domain/common';
-import AddressAmountView, {
-  AddressAmountProps,
-} from '../../../presentation/wallet/send/address-amount';
+import type { RootReducerState } from '../../../domain/common';
+import type { AddressAmountProps } from '../../../presentation/wallet/send/address-amount';
+import AddressAmountView from '../../../presentation/wallet/send/address-amount';
 import { selectBalances } from '../selectors/balance.selector';
-import { selectMainAccount } from '../selectors/wallet.selector';
+import {
+  selectAllAccountsIDsSpendableViaUI,
+  selectMainAccount,
+} from '../selectors/wallet.selector';
 
 const mapStateToProps = (state: RootReducerState): AddressAmountProps => ({
-  account: selectMainAccount(state),
+  changeAccount: selectMainAccount(state),
```

Comment:
not clear why now we calling it `changeAccount` (need to finish my review yet) 

How do we know the current "active" account?

Reply:
When u send coin from popup, after selection the asset to send. Marina will select utxos from all accounts. So if u own LBTC on a covenant account & on the main account, u are not able to "select" your utxos (or filter utxos by account(s)). That's why if change is needed, the main account is used.

In my mind this is a temporary solution. I think the user should be able to select the "change account" during "select asset fees" step.

I've renamed the variable because that account is the one used to generate change addresses (including taxi one).